src/chrome/chromeDebugAdapter.ts

```diff
@@ -376,6 +384,26 @@ export abstract class ChromeDebugAdapter implements IDebugAdapter {
         this._chromeConnection.onClose(() => this.terminateSession('websocket closed'));
     }
 
+    private runAndMeasureProcessingTime(notificationName: string, procedure: () => Promise<void>): Promise<void> {
+        const startTime = Date.now();
+        const startTimeMark = process.hrtime();
+        let properties: IExecutionResultTelemetryProperties = {
+            startTime: startTime.toString()
+        };
+
+        return procedure().then(() => {
+            properties.successful = 'true';
+        }, (e) => {
+            properties.successful = 'false';
+            properties.exceptionType = "generic";
```

Comment:
Is there a more intuitive way to put this information? If I read generic in this field, I don't think I'll understand what it means

Reply:
How about not setting this field?

Reply:
Renamed to 'firstChance'